extensions/wikia/GlobalShortcuts/scripts/AddDefaultShortcuts.js

```diff
@@ -65,7 +65,7 @@ require(['GlobalShortcuts', 'PageActions', 'mw', 'wikia.loader'],
 						id: 'wikia:Search',
 						caption: mw.message('global-shortcuts-caption-search-for-a-page').plain(),
 						fn: function () {
-							$('#searchInput')[0].focus();
+							$('.wds-global-navigation').find('.wds-global-navigation__search-input')[0].focus();
```

Comment:
@kvas-damian added `id="globalNavigation"` in https://github.com/Wikia/app/pull/11281/files#diff-f850972c6a894a3d071e2a573d2aa230R2. I think we should do the same here.